templates/AccessLogs.sol.template

```diff
@@ -107,13 +107,18 @@ library AccessLogs {
         AccessLogs.Context memory a,
         Memory.PhysicalAddress readAddress
     ) internal pure returns (uint64) {
-        bytes8 b8 = a.buffer.consumeBytes8();
-        bytes32 valHash = keccak256(abi.encodePacked(b8));
+        bytes32 readData = a.buffer.consumeBytes32();
+        bytes32 valHash = keccak256(abi.encodePacked(readData));
+
+        Memory.PhysicalAddress leafAddress = readAddress.truncateToLeaf();
+        uint64 offset = readAddress.minus(leafAddress);
+        bytes8 readValue = bytes8(readData << (offset << 3));
```

Comment:
`bytes8(readData << (offset << 3))` this line is being used in both `readWord` and `writeWord`, can we make this a function to reduce duplicate? And define this `3` a constant somewhere?